manifest.json

```diff
@@ -6692,6 +6692,52 @@
                 }
             }
         },
+        "co.uk.AlexW-578.CloseConfirm": {
+            "name": "CloseConfirm",
+            "description": "This stops the close button on the window as well as alt-f4 from closing the game but rather opening up the dash to the exit screen.",
+            "category": "Misc",
```

Comment:
probably fits best in `Keybinds & Gestures`

Reply:
Sure, I wasn't 100% sure which category that they would fall into.
I'll change it to be that then